src/olympia/abuse/tests/test_admin.py

```diff
@@ -32,39 +32,40 @@ def setUpTestData(cls):
         cls.addon1.name.__class__.objects.create(
             id=cls.addon1.name_id, locale='fr', localized_string='Elu'
         )
-        cls.addon2 = addon_factory(guid='@guid2', name='Two')
-        cls.addon3 = addon_factory(guid='@guid3', name='Three')
+        cls.addon2 = addon_factory(guid='@guid2', name='Owt')
+        cls.addon3 = addon_factory(guid='@guid3', name='Eerth')
```

Comment:
Did you mean this to be `Eerht`? 😉

Reply:
Any string would do, as it's only really useful for debugging, but it should totally have been `Eerht` !

Reply:
https://github.com/mozilla/addons-server/pull/19504/commits/a44b303c5d7dd9b5318572f7b2d4f5e2e9787113